packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -2,7 +2,11 @@
     <a
         :aria-label="linkAltText"
         href="/"
-        :class="$style['c-logo']"
+        :aria-disabled="linkDisabled"
+        :class="[
+            $style['c-logo'],
+            { [$style['disabled']]: linkDisabled }
```

Comment:
Would it be better to disable the link by switching the `a` tag to a `span` if this is set (via `is-component` in Vue)

Reply:
Definitely. This is where me being relatively new to Vue comes through. Will attempt to update it now.